### PR TITLE
Require r-base>=3.5 for RSeqC Conda envs

### PR DESCRIPTION
**Pull request by c-mertes**
## PR checklist

This is the prerequisite to close the following bug: nf-core/rnaseq#685 

This PR mainly enforces a working conda env with the appropriate R version needed by rseqc

### CODE REVIEW

File: modules/rseqc/innerdistance/main.nf [hunk 1]
```diff
@@ -11,7 +11,7 @@ process RSEQC_INNERDISTANCE {
         mode: params.publish_dir_mode,
         saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
 
-    conda (params.enable_conda ? "bioconda::rseqc=3.0.1" : null)
+    conda (params.enable_conda ? "bioconda::rseqc=3.0.1 'conda-forge::r-base>=3.5'" : null)
```
Comment (drpatelh) [inline modules/rseqc/innerdistance/main.nf:14]:
```suggestion
    conda (params.enable_conda ? "bioconda::rseqc=3.0.1 conda-forge::r-base>=3.5" : null)
```

Comment (drpatelh) [reply modules/rseqc/innerdistance/main.nf:14]: Thanks @c-mertes ! I realise the other RSeQC modules are unaffected by this dependency but be great to update them too for consistency.

Comment (c-mertes) [reply modules/rseqc/innerdistance/main.nf:14]: Thanks for reviewing. It is added to the other ones.

The single ticks are required as it is pasted to the command line as plain text and hence, the `>` will through an error. It works with a pure `=`. At least on my Centos 8 installation.

Comment (drpatelh) [reply modules/rseqc/innerdistance/main.nf:14]: Ah, I didn't know that 👍🏽

**Pull request merged**